### PR TITLE
12.0 mig project_task_default_stage

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: project_task_default_stage/README.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* Carlos Dauden <carlos@incaser.es>
+* Sergio Teruel <sergio@incaser.es>
+* Duc, Dao Dong <duc.dd@komit-consulting.com> (https://komit-consulting.com)
+* `Tecnativa <https://www.tecnativa.com>`_:
+
```
Comment (emagdalenaC2i) [inline project_task_default_stage/README.rst:68]: Add contributor in the following format:
Ernesto Tejeda <youremail@tecnativa.com> (Tecnativa https://www.tecnativa.com)

File: project_task_default_stage/readme/CONTRIBUTORS.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+* Carlos Dauden <carlos@incaser.es>
+* Sergio Teruel <sergio@incaser.es>
+* Duc, Dao Dong <duc.dd@komit-consulting.com> (https://komit-consulting.com)
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * Ernesto Tejeda
```
Comment (emagdalenaC2i) [inline project_task_default_stage/readme/CONTRIBUTORS.rst:6]: Add contributors in the following format:
Ernesto Tejeda youremail@tecnativa.com (Tecnativa https://www.tecnativa.com)

**Review by emagdalenaC2i — APPROVED** [review]

File: project_task_default_stage/README.rst (hunk 1 above)
Comment (pedrobaeza) [reply project_task_default_stage/README.rst:68]: @emagdalenaC2i email is not mandatory and we prefer to not put it (people spam us asking for free support).

**Review by cristinamartinrod — APPROVED** [review]
Tested on runbot. LGTM 👍🏻

**Pull request merged**